### PR TITLE
fix(curriculum): rename shopping cart project for JS curriculum

### PR DESCRIPTION
**Pull request by jdwilkin4**
## Summary of changes

The JS team decided to rename the shopping cart project to "Learn basic OOP" instead of "Learn Data Strucutures"

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: client/src/pages/learn/2022/javascript-algorithms-and-data-structures/learn-data-structures-by-building-a-shopping-cart/index.md
Comment (naomi-lgbt) [inline client/src/pages/learn/2022/javascript-algorithms-and-data-structures/learn-data-structures-by-building-a-shopping-cart/index.md:1]: Believe the directory here needs a rename.

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by huyenltnguyen — APPROVED** [review]
LGTM.

I made some suggestions on the capitalization of the title. They aren't in the scope of the PR though, so feel free to punt on those.

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -374,8 +374,8 @@
           "In this statistics calculator project, you'll gain experience in handling user input, DOM manipulation, and method chaining by performing statistical calculations like mean, median, mode, variance, and standard deviation."
         ]
       },
-      "learn-data-structures-by-building-a-shopping-cart": {
-        "title": "Learn Data Structures By Building A Shopping Cart",
+      "learn-basic-oop-by-building-a-shopping-cart": {
+        "title": "Learn Basic OOP By Building A Shopping Cart",
```
[the remark this answers is not in the document]
Comment (jdwilkin4) [reply client/i18n/locales/english/intro.json:378]: Good call. This is in line with the other projects 
I'll change it 👍

**Comment by Sembauke** [comment]
@naomi-lgbt can you give this another look?

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**